docs/websocket.md

```diff
@@ -586,3 +586,154 @@ eg:
    }
 }
 ```
+
+### Trades
+
+The trades channel allows clients to receive all trades for the market. This is the fastest way to receive trades made on the market.
+
+#### Subscribing
+
+To subscribe send:
+
+```json
+{
+  "type": "subscribe",
+  "channel": "trades",
+  "id": "DAI-USDC"
+}
+```
+
+|Field Name|JSON type|Description|
+|----------|---------|-----------|
+|type|string|Must be set to "subscribe"|
+|channel|string|Must be set to "trades"|
+|id|string|The market to subscribe to. e.g. WETH-DAI, WETH-USDC, DAI-USDC|
+
+#### Initial Response
+
+The initial response will contain the entire state of the active market:
```

Comment:
nit: I think the initial response will probably return the most recent 100 trades